packages/react-native-platform-override/src/e2etest/FileSystemRepository.test.ts

```diff
@@ -119,7 +119,7 @@ test('deleteFile - Exists', async () => {
 
 test('deleteFile - Doesnt Exist', async () => {
   await usingFiles(['0.59.9/Icon-60@2x.conflict.png'], async repo => {
-    // @ts-ignore Typings don't know about rejects
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
     expect(repo.deleteFile('nope')).rejects.toThrow();
```

Comment:
We previously had conflicting typings for global `expect` between Jasmine and Jest. Looks like these packages weren't quite isolated, so the removal of Jasmine typings causes a different break here.